canvas_modules/harness/assets/styles/harness.scss

```diff
@@ -17,15 +17,23 @@
 @use "@carbon/react/scss/themes" as *;
 @import "./carbon.scss";
 
+
 // Switch the theme's below to change light <-> dark theme for harness
 // Modify common-canvas/src/themes/light.scss to switch common-canvas.
 $theme: $g10; // The light (gray 10) theme
 // $theme: $g90; // The dark theme
 
 :root {
-	@include theme();
+	@include theme($g10);
+}
+
+[data-carbon-theme='g90'] {
```

Comment:
These both should have double quotes.  It's going to fail the build.

Reply:
Yes, fixed these.